index.js

```diff
@@ -76,19 +78,26 @@ if (!output && !dir && !argv.replace) {
 // Use warn to avoid writing to stdout
 console.warn(chalk.bold.red(logo))
 
-spinner.text = `Loading Config`
+spinner.text = 'Loading Config'
 spinner.start()
-Promise.all([ globber(input), getConfig({}, argv.config) ])
-  .then((arr) => {
-    // Until parameter destructuring is supported
-    let files = arr[0]
+getConfig({}, argv.config)
+  .then(() => {
+    spinner.succeed()
 
-    if (!files || !files.length) throw new Error('You must pass a list of files to parse')
+    if (input && input.length) return globber(input)
+    // Read from stdin:
+    else {
+      if (!argv.output) throw new Error('Must specify --output when reading from stdin')
```

Comment:
I'm thinking about adding process.stdout support in a later PR.

The main reason for this line is to throw an error if `--dir` or `--replace` is passed, since we don't have a filename to write.